nbextensions/upload_it/main.js

```diff
@@ -79,14 +79,26 @@ define([
             window.console.log("Uploading ", notebook.notebook_path, " to ", url, formdata);
             $.ajax({
               url: url,
+              xhrFields: {withCredentials: true},
               data: formdata,
               contentType: false,
               processData: false,
               method: "POST",
               success: function(data, status, jqXHR) {
-                window.console.log("Upload OK", data);
+                // Open the report in a new tab.
+                var u = new URL(url);
+                var reportURL = u.protocol + "//" + u.host + data;
```

Comment:
ditto

Reply:
Thank you for the hint. Used  'u.pathname = data'.